src/libs/print_settings.c

```diff
@@ -0,0 +1,1398 @@
+/*
+    This file is part of darktable,
+    copyright (c) 2014-2015 pascal obry.
+
+    darktable is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    darktable is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with darktable.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <glib.h>
+
+#include "common/collection.h"
+#include "common/colorspaces.h"
+#include "common/image_cache.h"
+#include "common/styles.h"
+#include "common/variables.h"
+#include "common/cups_print.h"
+#include "common/image_cache.h"
+#include "common/pdf.h"
+#include "common/tags.h"
+#include "common/printprof.h"
+#include "dtgtk/resetlabel.h"
+#include "libs/lib.h"
+#include "gui/gtk.h"
+#include "gtk/gtk.h"
```

Comment:
that's `#include <gtk/gtk.h>` I guess? But since it gets pulled in by gui/gtk.h you can just omit it.